supereeg/brain.py

```diff
@@ -250,7 +252,8 @@ def update_info(self):
         self.n_elecs = self.data.shape[1] # needs to be calculated by sessions
         self.n_sessions = len(self.sessions.unique())
         try:
-            self.n_secs = np.true_divide(counts, np.array(sample_rate))
+            index, counts = np.unique(self.sessions, return_counts=True)
```

Comment:
Can you unpack the logic here a bit?  E.g. why is this in a try/except block rather than an if/else statement?

Reply:
ok-- can you add a comment (in the code) that this could be changed to an if/else statement?